src/main/java/com/example/webapp/Service/DepartmentService.java

```diff
@@ -0,0 +1,59 @@
+package com.example.webapp.Service;
+
+import com.example.webapp.Model.Department;
+import com.example.webapp.Repository.DepartmentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class DepartmentService {
+@Autowired
+DepartmentRepository departmentRepository;
+    public ResponseEntity<List<Department>> getAll(){
```

Comment:
not good to return is service ResponseEntity. You need to return List<Department>

Reply:
Resolved